change/@internal-storybook-a6d95240-9957-4057-a6ce-6d2a50b18088.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

Comment:
Unsure if this should be `prerelease` or `none`. It doesn't change anything in the library, but fixes the documentation for a beta feature.

Reply:
I like pre-release since its about fixing docs for a feature we haven't released yet. Is there value for having a new kind of changelog for doc-changes? like `doc-change` get visibility on when we are improving our storybook?

Reply:
Will stick to `prerelease` then. A new type is probably not worth it.